### PR TITLE
Removed more Eris stuff I missed.

### PR DESCRIPTION
**Pull request by SirSiggles**
## About The Pull Request

Making the rebase more lore complaint with our lore. And solved some issues from #12 

## Details

<details>
  <summary>Expand</summary>
Removed mentions of Asters, Moebius and Ironhammer I missed. With help from issue #12 on the last two. I also removed some NeoTheology references.
</details>

## Screenshots

<details>
  <summary>Expand</summary>

</details>

### CODE REVIEW

File: code/game/jobs/job/church.dm [hunk 1]
```diff
@@ -54,15 +54,15 @@
 	For those who are not part of the fold, the next best thing you can offer is a dignified funeral. The chapel area contains coffins and machinery to commit the dead unto the void. Burial at space. Any player who is given a proper funeral will have their respawn time reduced, allowing them to rejoin the crew as a new character more quickly, after death."
 
 
-	duties = "Represent the interests of NT disciples aboard Eris. Protect them from persecution and speak for them.<br>\
+	duties = "Represent the interests of NT disciples aboard the NEV NOrthern Light. Protect them from persecution and speak for them.<br>\
```
Comment (EvilJackCarver) [inline code/game/jobs/job/church.dm:57]: Shift key hung

File: code/game/jobs/job/engineering.dm [hunk 2]
```diff
@@ -116,7 +116,7 @@ Eris is your home, your life, and your livelihood. Take pride in it, and in your
 
 	loyalties = "	As an engineer, your first loyalty is to your fellow engineers. Ensure they are safe and well supplied, defend them, assist them, and share everything with them. If problems arise between you, ask the Chief Engineer to rule on it. Don't snitch on your fellow clanmates by calling Aegis. The Chief Engineer is the head of your clan, and in many ways a father figure. Trust in their wisdom and follow their instructions above anyone else's.<br>\
 	<br>\
-	Your second loyalty is to your ship. Unlike most of your crew, who would simply go home, Eris IS your home. Avoid abandoning ship at all costs. This is where you live and it's worth sacrificing for. Take pride in your work, and make Eris something to be cherished."
+	Your second loyalty is to your ship. Unlike most of your crew, who would simply go home, the Northern Lgiht IS your home. Avoid abandoning ship at all costs. This is where you live and it's worth sacrificing for. Take pride in your work, and make the Northern Light something to be cherished."
```
Comment (EvilJackCarver) [inline code/game/jobs/job/engineering.dm:119]: Minor typo

File: code/game/jobs/job/security.dm [hunk 3]
```diff
@@ -236,15 +236,15 @@
...
 	<br>\
 	You are a professional soldier and a hardened mercenary, no stranger to violence. You are required to employ your talents in order to bring an end to threats and conflict situations. As a consummate professional, you're often expected to put your pride aside, and work with others. Tactics and teamwork are vital.<br>\
 	<br>\
-	You are paid to act, not to think. When in doubt, follow orders, and leave the hard choices to someone else. Trust in your chain of command. Remember that you are the lowest rank in ironhammer, and you report to everyone else in your organisation. Inspector, medspec, gunnery sergeant and commander, are all your superior officers, their orders should be obeyed.<br>\
+	You are paid to act, not to think. When in doubt, follow orders, and leave the hard choices to someone else. Trust in your chain of command. Remember that you are the lowest rank in Aegis, and you report to everyone else in your organisation. Inspector, Medspec, gunnery sergeant and Commander, are all your superior officers, their orders should be obeyed.<br>\
```
Comment (EvilJackCarver) [inline code/game/jobs/job/security.dm:243]: Probably would do better in terms of grammar if it was 'The Inspector, ...' in the end sentence here

File: code/game/objects/items/weapons/tools/misc.dm [hunk 4]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/tool/omnitool
-	name = "Asters \"Munchkin 5000\""
-	desc = "Fuel powered monster of a tool. Its weldier part is most advanced one, capable of welding things without harmfull glow and sparks, so no protection needed."
+	name = "\"Munchkin 5000\""
+	desc = "A Fuel powered monster of a tool. Its weldier part is most advanced one, capable of welding things without harmfull glow and sparks, so no protection needed."
```
Comment (EvilJackCarver) [inline code/game/objects/items/weapons/tools/misc.dm:3]: Can we get a hyphen in "fuel-powered"?

File: code/game/jobs/job/engineering.dm (hunk 2 above)
Comment (SirSiggles) [reply code/game/jobs/job/engineering.dm:119]: Where's the typo? Don't see it right now.

Comment (EvilJackCarver) [reply code/game/jobs/job/engineering.dm:119]: >... the Northern **Lgiht** ...

**Comment by EvilJackCarver** [comment]
Speedmerging

**Pull request merged**